05-specifying-sample-designs.Rmd

```diff
@@ -1 +1,705 @@
+
+```{r}
+#| label: srs2
+#| eval: FALSE
+des_srs2 <- dat %>%
+  as_survey_design()
+```
+
+If some post-survey adjustments were implemented and the weights are not all equal, specify the design as:
+
+```{r}
+#| label: srs3
+#| eval: FALSE
+des_srs3 <- dat %>%
+  as_survey_design(weights = wtvar, fpc = fpcvar)
+```
+
+where `wtvar` is the variable for the weight on the data. Again, the fpc can be omitted if it is not necessary because the frame is large.
+
+#### Example {-} 
+
+The {survey} package in R provides some example datasets to use and those will be used throughout this chapter. The Academic Performance Index (API) was a program administered by the California Department of Education. These datasets are derived from that data and include a population file (frame) of all schools with at least 100 students and several samples from that data. A SRS of 200 schools was sampled and included in `apisrs`. The SRS sample data is illustrated below:
+
+```{r}
+#| label: apisrs_display
+options(tidyverse.quiet = TRUE)
+library(tidyverse)
+library(survey)
+library(srvyr)
+
+data(api)
+
+apisrs_slim <- 
+  apisrs %>%
+  as_tibble() %>%
+  arrange(dnum, snum) %>%
+  select(cds, dnum, snum, dname, sname, fpc, pw)
+
+```
+
+School districts have identifiers `dnum` within counties and schools have identifiers of `snum` within districts along with their names, `dname` and `sname`. The unique identifier for a school is `cds` which is unique within the state. Additionally, the `fpc` is included as `fpc` and the weight as `pw`. This design should be specified as:
+
+```{r}
+#| label: apisrs_des
+des_apisrs <- apisrs_slim %>%
+  as_survey_design(weights = pw, fpc = fpc)
+
+des_apisrs
+summary(des_apisrs)
+```
+
+Printing the design object above, the design is described as an "Independent Sampling design" which is another terminology for SRS. The ids are specified as `1` which means there is no clustering (a topic described later in this chapter), the fpc variable is indicated, and the weights are indicated. When looking at the summary of the design object, the population size is given a summary of the probabilities (inverse of the weights).
+
+### Simple random sample with replacement
+
+- Description: The simple random sample with replacement (SRSWR) is a sampling design where a sample is selected from a sampling frame with the units being replaced before drawing again so units can be selected more than once.
+- Requirements: Must have a sampling frame for entire population
+- Advantages: Simple, requires no information about the units apart from contact information
+- Disadvantages: Sampling frame may not be available for entire population. Not feasible for in-person data collection. Units can be selected more than once resulting in a smaller realized sample size. For small populations, SRSWR has larger standard errors than SRS designs.
+- Examples: Selecting items out of a hat but replacing them between each draw.
```

Comment:
Do we want to have a "real world" example for these.  Below you say that you had to create an example since one did not exist, so it might be good to help people understand why this would be used.  Or is that out of scope for this book?

Reply:
I've added an example which is still a bit unusual but I witnessed in practice.